--- ezpublish_legacy/kernel/setup/steps/ezstep_welcome.php.org	2014-09-05 11:06:31.718258749 +0000
+++ ezpublish_legacy/kernel/setup/steps/ezstep_welcome.php	2014-09-05 11:07:45.724225972 +0000
@@ -64,6 +64,11 @@
         $this->PersistenceList['tests_run'] = $testsRun;
         $this->PersistenceList['optional_tests_run'] = $testsRun;
 
+        if ( $this->hasKickstartData() )
+        {
+            return $this->kickstartContinueNextStep();
+        }
+
         return false; // Always show welcome message
     }
 
